src/components/WindRose/data.ts

```diff
@@ -0,0 +1,208 @@
+import { type ChartData } from 'react-windrose-chart';
```

Comment:
If this is mock, I would suggest moving data to the `mocks` directory and store it as a JSON object. You can then import data the same as you import components:

```ts
import data from '@/mocks/windrose/data.json';
```

You will have to cast data to proper type before using it as data loaded from JSON is of `any` type so you will have to use:

```ts
setChartData(data as ChartData[]);
```